packages/manager/modules/exchange/src/translations/Messages_fr_FR.json

```
@@ -1249,23 +1261,24 @@
   "exchange_ACTION_sending": "Envoi en cours...",
   "exchange_ACTION_configure_help_title": "Configuration service Exchange",
   "exchange_ACTION_configure_help_lockoutThreshold_title": "Seuil de verrouillage du compte",
-  "exchange_ACTION_configure_help_lockoutThreshold_text": "Ce paramètre de sécurité détermine le nombre de tentatives d'ouverture de session infructueuses qui provoque le verrouillage d'un compte utilisateur. Un compte verrouillé ne peut pas être utilisé jusqu'à sa réinitialisation par un administrateur ou jusqu'à ce que la durée de verrouillage du compte soit expirée. Vous pouvez définir une valeur comprise entre 1 et 14 tentatives d'ouverture de session infructueuses. Si vous définissez la valeur sur 0, le compte ne se verrouille jamais.",
+  "exchange_ACTION_configure_help_mfa_text": "<p>Cette fonctionnalité vous permet d'activer ou de désactiver la double authentification pour <strong>l'ensemble de votre service Exchange</strong>.</p><p>L'utilisateur final devra télécharger une application d'authification (Microsoft Authenticator, Google Authenticator, Authy...), scanner le QR code affiché à la prochaine connexion à son compte exchange, puis saisir le code fourni.</p><p>Une fois la fonctionnalité activée sur votre service, vous pourrez l'activer sur les comptes désirés dans la section comptes e-mail.</p>",
```

Couldn't be splitted into multiple keys (to avoid any `<p>` tag element for instance)?
It could be applied to the whole file that contains large translations, Isn't it?

- :warning: `d'authification` > `d'authentification`
- `...` > `…`

```suggestion
  "exchange_ACTION_configure_help_mfa_text": "<p>Cette fonctionnalité vous permet d'activer ou de désactiver la double authentification pour <strong>l'ensemble de votre service Exchange</strong>.</p><p>L'utilisateur final devra télécharger une application d'authentification (Microsoft Authenticator, Google Authenticator, Authy…), scanner le QR code affiché à la prochaine connexion à son compte exchange, puis saisir le code fourni.</p><p>Une fois la fonctionnalité activée sur votre service, vous pourrez l'activer sur les comptes désirés dans la section comptes e-mail.</p>",
```